bolt/users_test.go

```diff
@@ -21,7 +20,7 @@ var cmpOptions = cmp.Options{
 func TestUsersStore_Get(t *testing.T) {
```

Comment:
1. could you add tests for `Get` to cover search via Name and Provider, and the reasonable errors?
1. could you add test coverage for `each`?

Reply:
Added